lib/kracken/controllers/json_api_compatible.rb

```diff
@@ -0,0 +1,45 @@
+module Kracken
+  module Controllers
+    module JsonApiCompatible
+      def self.included(base)
+        base.instance_exec do
+          before_action :munge_chained_param_ids!
+          skip_before_action :verify_authenticity_token
+
+          unless Rails.env.development? # ZOMG
+            rescue_from StandardError do |error|
+              render_json_error 500, error
+            end
+
+            rescue_from ActionController::RoutingError do |error|
+              render_json_error 404, error
+            end
+
+            rescue_from ActiveRecord::RecordNotFound do |error|
+              render_json_error 404, error
+            end
+          end
```

Comment:
Makes it harder to debug locally if we have better errors installed.

I also added a check to see if `ActiveRecord` is defined, since we may not be using it (and don't in the dummy app in the specs)